src/Game/Player.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react"
+
+export default function Player({id, name, symbol, updatePlayerTitle}) {
+
+    const [isEditing, setIsEditing] = useState(false)
+
+    const handleEdit = () => {
+        if (name === '') {
+            alert('Name cannot be empty');
+        } else {
+            setIsEditing((editing) => !editing);
+        }
+    }
+
+    const handlePlayerNameChange = (value) => {
+        updatePlayerTitle(id, value)
+    }
+
+    let playerName = <span className="player-name">{name}</span>
+    if(isEditing) {
+        playerName = <input className="player-name" value={name} onChange={ (e) => handlePlayerNameChange(e.target.value) }></input>
+    }
+
+    return (
+        <li>
+            <span className="player">
+                {playerName}
+                <span className="player-symbol">{symbol}</span>
+            </span>
+            <button onClick={handleEdit}> {isEditing ? 'Save' : 'Edit'} </button>
+        </li>
+    )
+}
```

Comment:
<img width="855" alt="Screenshot 2023-11-27 at 10 12 11 PM" src="https://github.com/MuhammadAwaisGondal/TIcTacToe/assets/150359324/cbca2c5e-1af5-4dd5-a016-46696b5516f0">
I have set player one name as Awais and player 2 as Gondal but these changes are not implemented to "Next player is"... it is still showing as X and O

Reply:
After editing, click Save to save the player name.